src/Integration/Language.cs

```diff
@@ -127,13 +115,8 @@ public static Language ForProject(EnvDTE.Project dteProject)
                 throw new ArgumentNullException(nameof(dteProject));
             }
 
-            Guid projectKind;
-            if (!Guid.TryParse(dteProject.Kind, out projectKind))
-            {
-                return Unknown;
-            }
-
-            return KnownLanguages.FirstOrDefault(x => x.ProjectType == projectKind) ?? Unknown;
+            // TODO: remove this method - change the callers to call the following method directly
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)